docs/environment-setup.md

```diff
@@ -88,28 +88,32 @@ sudo add-apt-repository \
    1. Create the Docker group: `groupadd docker`
    1. Add your user to the group: `sudo usermod -aG docker $USER`
    1. Log out and log back in (you may have to restart if you are running through a virtual machine)
-9. Verify your installation by running `docker run hello-world`. This should print a hello world paragraph that includes a confirmation that Docker is working on your system.
-   _NOTE: This may cause errors if you have already tried to run docker before. If you get errors then run the following commands to reset it:_
+9. Make sure to you have docker running on your machine, you can start docker through the following methods
```

Comment:
One tip for Markdown: you can leave the numbers as `1.` and it will figure out what they should be automatically in the rendered form (i.e. you don't need to use 9., 10., etc)

Reply:
That's great! Must have been a facepalm moment when you saw the number change in the other steps lol. Thanks David!

Reply:
No facepalms from me, this stuff takes time to learn.  You're doing great.